zeppelin-web/src/app/notebook/notebook.controller.js

```diff
@@ -62,6 +62,10 @@ function NotebookCtrl ($scope, $route, $routeParams, $location, $rootScope,
   $scope.currentRevision = 'Head'
   $scope.revisionView = isRevisionPath($location.path())
 
+  $scope.$watch('note', function (value) {
+    $rootScope.pageTitle = value ? value.name : ''
+  }, true)
+  
```

Comment:
When I built on `zeppelin-web` in my local, a error is occurred like the following contents.
```
[INFO] /home/sora/zeppelin/zeppelin/zeppelin-web/src/app/notebook/notebook.controller.js
[INFO]    57:34  warning  Unnecessary escape character: \/  no-useless-escape
[INFO]    57:44  warning  Unnecessary escape character: \/  no-useless-escape
[INFO]    57:59  warning  Unnecessary escape character: \/  no-useless-escape
[INFO]    57:69  warning  Unnecessary escape character: \/  no-useless-escape
[INFO]    68:1   error    Trailing spaces not allowed       no-trailing-spaces
[INFO]   450:1   error    Trailing spaces not allowed       no-trailing-spaces
``` 
could you resolve this 68 line?
I guess the 450 line and your CI errors are related 68 line, so you can just remove space in 68 line.